packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -299,6 +301,7 @@ export function TransferPanelMain({
   const [to, setTo] = useState<L1Network | L2Network>(externalTo)
 
   const [loadingMaxAmount, setLoadingMaxAmount] = useState(false)
+  const [withdrawOnlyDialog, setWithdrawOnlyDialog] = useState(false)
```

Comment:
We actually have a neat `useDialog` hook that can handles this internally:

```
const [withdrawOnlyDialogProps, openWithdrawOnlyDialog] = useDialog()

...

<Dialog {...withdrawOnlyDialogProps} />

...

<button className="arb-hover underline" onClick={openWithdrawOnlyDialog}>
  Learn more.
</button>
```

It's used a couple of times in the codebase, so check out the other uses.